src/apollo/retry-link.js

```diff
@@ -1,14 +1,13 @@
 import { RetryLink } from 'apollo-link-retry'
 
 const retryLink = new RetryLink({
-  delay: {
-    initial: 500,
-    max: Infinity,
-    jitter: true
-  },
   attempts: (count, operation, error) => {
-    const max = operation.getContext().maxAttempts || 10000
-    return !!error && count < max && operation.getContext().isRetriable
+    const { isRetriable, maxAttempts } = operation.getContext()
+    const max = maxAttempts || 100
```

Comment:
`100` feels like a potential DDoS. Maybe to lower it to `10`?

Reply:
agree